submissions/career/career.md

```diff
@@ -0,0 +1,53 @@
+- The introduction works great, but imo doesn't really connect to the next paragraph, but ends quite abruptly
```

Comment:
Past Conor (?) basically said what I was going to say XD

Reply:
Lol this was nano's feedback which I didn't wanna lose at the time :p